includes/class-wc-payments-account.php

```diff
@@ -187,6 +187,74 @@ public function get_is_live() {
 		return ! empty( $account ) && isset( $account['is_live'] ) ? $account['is_live'] : null;
 	}
 
+	/**
+	 * Gets the various anti-fraud services that must be included on every WCPay-related page.
+	 *
+	 * @return array Assoc array. Each key is the slug of a fraud service that must be incorporated to every page, the value is service-specific config for it.
+	 */
+	public function get_fraud_services_config() {
+		$account = $this->get_cached_account_data();
+		if ( empty( $account ) || ! isset( $account['fraud_services'] ) ) {
+			// This was the default before adding new anti-fraud providers, preserve backwards-compatibility.
+			return [ 'stripe' => [] ];
+		}
```

Comment:
This is a tough one as it comes down to where the fraud keys are served from. If we moved it out into a separate endpoint we could call it when the plugin is activated, but I'm not sure if this will give us much more data and I'm not sure how valuable the first settings page visit would be before the merchant goes through the KYC. It may be a critical point. 

I think for now having some data is better than none so I'm leaning towards adding this if we need that pre-sign-up signal.